CMakeLists.txt

```diff
@@ -297,7 +297,7 @@ if(WIN32)
 endif()
 
 # check SSL libraries
-# TODO support GNUTLS, POLARSSL, CYASSL
+# TODO support GNUTLS, CYASSL
```

Comment:
Maybe make that "GnuTLS" (casing) and "wolfSSL" (as we no longer use the name cyassl anywhere)

Reply:
Ta - I saw that and meant to do it. I really should stop trying to do 3 or more things at once!